trestle/core/markdown/markdown_node.py

```diff
@@ -158,6 +188,29 @@ def _build_tree(
         md_node.subnodes = node_children
         return (md_node, i)
 
+    def _modify_header_level(self, header: str, level: int) -> str:
+        """Modify header level by specified level."""
+        if level == 0:
```

Comment:
I would call this "delta_level" or just delta to emphasize it represents a change of level and not a level.

Reply:
Good suggestion, I think this name will add more clarity. Added